File: CentristTraveler/Models/Post.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace CentristTraveler.Models
{
    public class Post : BaseModel
    {
        public int PostId { get; set; }
        public string Title { get; set; }
        public string Body { get; set; }
        public string ThumbnailPath { get; set; }
        public string BannerPath { get; set; }
        public string BannerText { get; set; }
        public string PreviewText { get; set; }
        public int CategoryId { get; set; }
        public List<Tag> Tags { get; set; }
    }
}

File: CentristTraveler/ClientCentristTraveler/src/app/models/post.ts
import { Tag } from './tag';

export class Post {
  id: number;
  title: string;
  body: string;
  thumbnail_path: string;
  preview_text: string;
  banner_path: string;
  banner_text: string;
  category_id: number;
  tags: Tag[];
  author_display_name: string;
  author_username: string;
}
